### PR TITLE
I added that the government should be re-organised to a bottom-up structure rather than a top-down. 

### PR DESCRIPTION
**Pull request by cowlesz**
I also added the free donuts every Saturday to see how far people can take this or if there are any editorial controls.

### CODE REVIEW

**Comment by Floppy** [comment]
Agree with the bottom-up, but we might need to (sadly) lose the free donuts :)

**Comment by Floppy** [comment]
I prefer my donuts on Friday anyway.

**Comment by cowlesz** [comment]
Hi James, 
Thanks for getting back to me, and sorry for the stupid donuts thing. 
Actually, I'm an MA Journalism, War and International Human Rights student currently on work placement at Peace News. I'd be interested in talking to you about your OpenPolitics project. 
Do you have another email address or is this the one you use regularly?
Cheers,  

Ben CowlesFreelance journalist / MA Journalism, War and International Human Rights Student(+44) 07580 051055@Cowleszasiapundits.com / www.10mag.com / monkeyboygoes.com



Date: Tue, 7 Apr 2015 03:39:33 -0700

**Comment by Floppy** [comment]
I'll drop you a line! If you fancy removing the donuts line, I imagine the bottom-up bit will go in. You can edit by clicking the pencil icon on the https://github.com/openpolitics/manifesto/pull/316/files page.

**Comment by digitalWestie** [comment]
Sounds good to me!

**Comment by philipjohn** [comment]
:+1: to bottoms up!

I'm more of an free donuts everyday man, though.

**Comment by yellowgopher** [comment]
So, do we need to enhance the role of parish, district and county councils (i.e. reverse what seems to be happening, despite things like the localism act)?

**Comment by digitalWestie** [comment]
Do we have something about being able to trigger referendums?

**Comment by yellowgopher** [comment]
I think there are a number of separate streams going on about how we redefine democracy and how that would work in a new constitutional arrangement...!  Needs to be tidied up?

**Comment by Floppy** [comment]
@cowlesz I suspect this will merge in happily if you remove the donut line. Click edit on https://github.com/openpolitics/manifesto/pull/316/files to do that, and we can get it in. :)

File: manifesto/index.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ We believe that government should be:
 * Rational, with decisions and policy based on sound evidence, not media pressure.
 * Courageous, able to make difficult decisions without pandering to short-termism.
 * Ethical: dedicated to creating a sustainable society and economy with progressive moral values.
+* Bottom-up: People should be telling the government what they want from a local level, rather than being told from the top. 
+* Give the people free donuts every Saturday
```
Comment (cowlesz) [inline manifesto/index.md]: Okay, let's delete the donuts thing.

**Comment by Floppy** [comment]
:+1:

**Comment by digitalWestie** [comment]
If this means we move towards a con-federal model I'm all for it :+1:

**Comment by Floppy** [comment]
This proposal is open for discussion and voting. If you are a [contributor](https://votebot.openpolitics.org.uk/proposals/) to this repository (and not the proposer), you may vote on whether or not it is accepted. 

## How to vote
Vote by entering one of the following symbols in a comment on this pull request. Only your last vote will be counted, and you may change your vote at any time until the change is accepted or closed.

|vote|symbol|type this|points|
|--|--|--|--|
|Agree|:thumbsup:|`:thumbsup:`|1|
|Abstain|:hand:|`:hand:`|-1|
|Block|:thumbsdown:|`:thumbsdown:`|-1000|

Proposals will be accepted and merged once they have a total of 2 points when all votes are counted. Votes will be open for a minimum of 7 days, but will be closed if the proposal is not accepted after 90.

Votes are counted [automatically here](https://votebot.openpolitics.org.uk/proposals/316), and results are set in the merge status checks below.

## Changes

If the proposer makes a change to the proposal, no votes cast before that change will be counted.

**Pull request merged**